### ISSUE TITLE
Unhandled Exception during initialize

### ISSUE THREAD

**Issue by natintosh** [smileidentity/ios#209]
```
SmileID/SmileID.swift:95: Fatal error: 'try!' expression unexpectedly raised an error: Swift.DecodingError.keyNotFound(CodingKeys(stringValue: "prod_url", intValue: nil), Swift.DecodingError.Context(codingPath: [], debugDescription: "No value associated with key CodingKeys(stringValue: \"prod_url\", intValue: nil) (\"prod_url\").", underlyingError: nil))
Lost connection to device.
```

**Comment by jumaallan** [comment]
@natintosh please confirm your `smile_config.json` is correct or added on the project

Do not paste the contents here though 😄 It should have all tbe keys filled when you download it

**Comment by natintosh** [comment]
test_url and prod_url were missing from the config we downloaded. But still, the exception wasn't properly handled and caused a crash. This shouldn't be the expected behaviour, if a key is missing, I think iOS SDK should gracefully handle while sharing a helping information than it rather causing a crash

**Comment by jumaallan** [comment]
That will be fixed. Thank you for raising the issue @natintosh

**Comment by jumaallan** [comment]
Fixed on #216
